dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -223,215 +140,131 @@ private static Link getLinkFields(Map<String, String> map) throws ParseException
-        }
-
-        container.setFriendlyName(map.get(FRIENDLY_NAME));
-
-        if (map.get(MAX_CONTENTLETS) != null && !map.get(MAX_CONTENTLETS).isEmpty()){
-            container.setMaxContentlets(Integer.parseInt(map.get(MAX_CONTENTLETS)));
-        }
-
-        if (map.get(USE_DIV) != null && !map.get(USE_DIV).isEmpty()) {
-            container.setUseDiv(Boolean.parseBoolean(map.get(USE_DIV)));
-        }
-
-        if (map.get(STATICIFY) != null && !map.get(STATICIFY).isEmpty()) {
-            container.setStaticify(Boolean.parseBoolean(map.get(STATICIFY)));
-        }
-
-        container.setSortContentletsBy(map.get("sort_contentlets_by"));
-
-        container.setLuceneQuery(map.get("lucene_query"));
-
-        container.setNotes(map.get("notes"));
-
-        container.setIdentifier(map.get(IDENTIFIER));
+        container.setInode((String) map.get(INODE));
+        container.setOwner((String) map.get(OWNER));
+        container.setIDate((Date) map.get(IDATE));
+        container.setCode((String) map.get("code"));
+        container.setPreLoop((String) map.get("pre_loop"));
+        container.setPostLoop((String) map.get("post_loop"));
+        container.setShowOnMenu((Boolean) map.getOrDefault(SHOW_ON_MENU,false));
+        container.setTitle((String) map.get(TITLE));
+        container.setModDate((Date) map.get(MOD_DATE));
+        container.setModUser((String) map.get(MOD_USER));
+        container.setSortOrder((Integer) map.getOrDefault(SORT_ORDER,0));
+        container.setFriendlyName((String) map.get(FRIENDLY_NAME));
+        container.setMaxContentlets((Integer) map.getOrDefault(MAX_CONTENTLETS,0));
+        container.setUseDiv((Boolean) map.getOrDefault(USE_DIV,false));
+        container.setStaticify((Boolean) map.getOrDefault(STATICIFY,false));
+        container.setSortContentletsBy((String) map.get("sort_contentlets_by"));
+        container.setLuceneQuery((String) map.get("lucene_query"));
+        container.setNotes((String) map.get("notes"));
+        container.setIdentifier((String) map.get(IDENTIFIER));
         return container;
     }
 
-    public static List<Template> convertDotConnectMapToTemplate(final List<Map<String,String>> results)
-            throws ParseException {
+    public static List<Template> convertDotConnectMapToTemplate(final List<Map<String, Object>> results) throws ParseException {
 
         List<Template> ret = new ArrayList<>();
 
-        if(results != null && !results.isEmpty()){
-            for (Map<String, String> map : results) {
+        if (results != null && !results.isEmpty()) {
+            for (Map<String, Object> map : results) {
                 ret.add(getTemplateFields(map));
             }
         }
         return ret;
     }
 
     @NotNull
-    private static Template getTemplateFields(Map<String, String> map) throws ParseException {
+    private static Template getTemplateFields(Map<String, Object> map) throws ParseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.text.ParseException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)